sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionDescription.java

```diff
@@ -88,6 +88,12 @@
 @Retention(RetentionPolicy.RUNTIME)
 public @interface ExpressionDescription {
     String usage() default "";
+    /**
+     * @deprecated This field is deprecated as of Spark 3.0. Use {@link #arguments},
```

Comment:
Like `Scala-2.11` deprecation, can we deprecate this in Spark 2.4.4 since Spark 2.4.4 will be delivered before Spark 3.0.0?
cc @gatorsmile , @srowen , too.

Reply:
I don't have a strong opinion, but generally think it's fine to deprecate things we know we want to deprecate sooner than later.

Reply:
I don't have a strong opinion too. But the next release will be likely 3.0 before 2.4.4. It would be a bit odd the later release deprecate something before earlier versions.

Reply:
I don't have a strong opinion too. This deprecation isn't that important about which version we should drop.